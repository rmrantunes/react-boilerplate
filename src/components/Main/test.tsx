import { screen, render } from '@testing-library/react'
import { Main } from '.'

describe('<Main />', () => {
  it('Should render heading', async () => {
    render(<Main />)

    const heading = await screen.findByRole('heading', {
      name: /react avan√ßado/i
    })
    expect(heading).toBeInTheDocument()
    expect(heading).toMatchSnapshot()
  })
})
