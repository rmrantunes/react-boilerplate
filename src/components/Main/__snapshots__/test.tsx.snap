// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should render heading 1`] = `
<h1
  class="styles__Title-sc-dzx1ao-2 igfAWE"
>
  React avan√ßado
</h1>
`
